backend/tracim_backend/lib/rq/__init__.py

```diff
@@ -1,10 +1,19 @@
 # -*- coding: utf-8 -*-
+import enum
+
 import redis
 import rq
 
 from tracim_backend.config import CFG
 
 
+class RqQueueName(str, enum.Enum):
+    DEFAULT = "default"
```

Comment:
what is the role of the default queue ?

Reply:
A RQ worker uses this queue by default, it was the default argument of `get_rq_queue`.

Reply:
ok. understood.